### ISSUE TITLE
PanGestureHandler for web

### ISSUE THREAD

**Issue by MrLoh** [kmagiera/react-native-gesture-handler#540]
@jaulz thanks for the initial web support. I saw that unfortunately support for pan gestures is still missing. How feasible do you think it is to add support via RN-web's PanResponder? I'd be willing to help, but I'm unfamiliar with this so far.

**Comment by jaulz** [comment]
I don't think it'a big deal. You just need to listen to the `PanResponder` events and trigger the `react-native-gesture-handler` events if the conditions are met. Not complicated but you need to fiddle around with it I think which could take some time.

**Comment by Aleksion** [comment]
@MrLoh have you been taking a stab at this?

**Comment by MrLoh** [comment]
No, haven’t had any time for it unfortunately.

**Comment by maxired** [comment]
For anyone coming here, PanGestureHandler for web is implemented in https://github.com/kmagiera/react-native-gesture-handler/pull/555
I can confirm the work on a browser.

To have it working,I got to defined a `minDist` attribute  to my `PanGestureHandler`.
I don't think this is intended as we create  to minDistSqrt https://github.com/kmagiera/react-native-gesture-handler/blob/5713701f4359dc11f4fb7ad3e4d78100b49730ce/web/PanGestureHandler.js#L72, but `shouldActivateUnderCustomCriteria` is called with props and not this default config, so I guess that make sense.


```javascript
  <PanGestureHandler minDist={3} onGestureEvent={this._onPanGestureEvent} >
```

**Comment by EvanBacon** [comment]
@maxired could you open a new issue with the details of the parity bug.

**Comment by kmagiera** [comment]
Thanks to @EvanBacon we recently merged #555 that brings gesture handler to web. Closing this. In case of any issues please open a new ticket
